### PR TITLE
LRDOCS-8181 App Builder Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
https://issues.liferay.com/browse/LRDOCS-8181

@rbohl

/cc @rafaprax @jeyvison @matheus-xavier

- LRDOCS-8181 comments and editing
- LRDOCS-8181 rename to be more specific
- LRDOCS-8181 update and edit the "creating a standard app" article
- LRDOCS-8181 edits and nav
- LRDOCS-8181 drafting and drawing
- LRDOCS-8181 nav work, re-organinzing
- LRDOCS-8181 add the table view as a workflow management option
- LRDOCS-8181 add info about assigning a task if accessed from the notification
- LRDOCS-8181 removing comments and incorporating SME review
- LRDOCS-8181 Wordsmithing.
- LRDOCS-8181 Wordsmithing.
- LRDOCS-8181 Wordsmithing.
- LRDOCS-8181 Wordsmithing.
- LRDOCS-8181 Wordsmithing.

### CODE REVIEW

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md [hunk 1]
```diff
@@ -19,61 +19,55 @@ Why use App Builder when the [Liferay Forms](../../process-automation/forms/user
...
-![App Builder widgets are placed on Liferay DXP pages.](./app-builder-overview/images/03.png)
+![App Builder widgets are placed on pages.](./app-builder-overview/images/03.png)
 
-**Standalone:** Deploy the application to a dedicated page that's not part of a site and not visible to any navigation menu. This limits access to only people with the link.
+**Standalone:** Deploy the application to a dedicated page that's not part of a site and not visible to any navigation menu. This limits access to only those with the link.
 
 ![Standalone App Builder applications are accessible only by direct link](./app-builder-overview/images/04.png)
 
-**Product Menu:** Deploy the application to the Control Panel, the Site Menu, or both. This gives you flexibility in deciding where administrative Users will access the application.
+**Product Menu:** Deploy the application to the Control Panel, the Site Menu, or both. This gives you flexibility in deciding which administrative Users can access the application.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md:64]: @rbohl -- Given that we have a "Global Menu" now that has submenus for Applications / (Commerce) / Control Panel, do you think this description is clear enough for users on where things will be deployed?

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-the-role-change-request-object.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+![The Role Change Request object flows through several steps in this Workflow Powered App.](./creating-a-workflow-powered-application/drawings/role-request-app.png)
+
+Here you'll create the backing object, form views, and table views. See [Creating the Role Change Request App](./creating-the-role-change-request-app.md) to create the Workflow Powered App itself.
+
+For detailed instructions on creating App Builder objects, form views, and table views, see [Creating a Standard Application](./creating-a-standard-application.md). The same object can back a Standard App or a Workflow Powered App. The difference is in how you create the app from the elements of the App Builder framework. You can create the object, form views, and table views from the Objects entry in Applications Menu (![Applications Menu](../../images/icon-applications-menu.png)) &rarr; Applications &rarr; App Builder.
+
+#### **Object:** Create the object named _Role Change Request_.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-the-role-change-request-object.md:13]: @rbohl any particular reason why these are `####` vs. `##` ?

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md (hunk 1 above)
Comment (rbohl) [reply docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md:64]: No probably not

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-the-role-change-request-object.md (hunk 2 above)
Comment (rbohl) [reply docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-the-role-change-request-object.md:13]: Probably the result of a reorg--I'm fine with second level

**Comment by jrhoun** [comment]
Merged. Thanks for your work on this @rbohl. Minor changes here: https://github.com/liferay/liferay-learn/commit/bd8e2a7e1e8b55ca3018d3fb18204bf1e7b24f60

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md (hunk 1 above)
Comment (rbohl) [reply docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md:64]: https://issues.liferay.com/browse/LRDOCS-8655

**Pull request closed without merging**